account_chart_update_multilang/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl-3.0)
+
+{
+    "name": "Account Chart Update Multilang",
+    "summary": "Update tax and fiscal position templates with multilang",
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "category": "Accounting",
+    "website": "https://github.com/OCA/account-financial-tools",
+    "author": "Moduon, Odoo Community Association (OCA)",
+    "maintainers": ["edlopen"],
+    "license": "AGPL-3",
+    "application": False,
```

Comment:
suggestion: make it auto-installable. If you happen to have all dependencies here, you probably want it.

```suggestion
    "auto_install": True,
```

(Before you ask, `application=False` is implicit)